src/components/topic-item/topic-item.jsx

```diff
@@ -0,0 +1,19 @@
+import {useNavigate} from 'react-router-dom';
+import './topic-item.css'
+
+
+
+const TopicItem = ({topic}) => {
+
+    const navigate = useNavigate()
+    const onNavigateHandler = () => navigate(`/topics/${topic.slug}`);
+
+    return (
+        <div className='topic-container' onClick={onNavigateHandler}>

```

Comment:
This onclick does not show the user that it is clickable, this may not be super accessible either. If you're going to use it this way, consider how you can show the user that the area is clickable. (hint, this is a css property)